### PR TITLE
Non-group functions test plan

### PR DESCRIPTION
**Pull request by Melirius**
Test plan for all the test fixes and updates in 4.17 that are not group-based (that were covered by https://github.com/KhronosGroup/SYCL-CTS/pull/400).

### CODE REVIEW

**Review by gmlueck — APPROVED** [review]

File: test_plans/individual-functions.asciidoc [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+=== 4.17.6. Integer functions
+
+`marray` tests will be added to every function from Table 178. _Integer functions which work on SYCL host and device,
+are available in the sycl namespace._ Additionally some problems in existing tests should be fixed and new tests added:
+
+* Reference implementation of `template <typename T, int N, typename R = typename std::make_unsigned<T>::type>
+sycl::vec<R, N> abs(sycl::vec<T, N> a)` should return `T`, not `R` following Table 178 definition.
+
+* Reference implementation of `template <typename T> T abs_diff(T a, T b)` should return `typename R = 
+typename std::make_unsigned<T>::type` instead following Table 178 definition. Implementation should be updated
```
Comment (keryell) [inline test_plans/individual-functions.asciidoc:243]:
```suggestion
typename std::make_unsigned_t<T>` instead following Table 178 definition. Implementation should be updated
```

File: test_plans/individual-functions.asciidoc [hunk 2]
```diff
@@ -0,0 +1,654 @@
...
+
+`marray` tests will be added to every function from Table 179. _Common functions which work on SYCL host and device,
+are available in the sycl namespace._ Additionally some problems in existing tests should be fixed and new tests added:
+
+* Reference implementations of `genfloat degrees(genfloat radians)`, `genfloat radians(genfloat degrees)`,
+`genfloat smoothstep(genfloat edge0, genfloat edge1, genfloat x)`, and `genfloat sign(genfloat x)`
+with `sycl::half` should be added.
+
+* Reference implementations of `genfloat max(genfloat x, genfloat y)` and `genfloat min (genfloat x, genfloat y)`
+should be fixed to provide correct undefined return value behavior (if any input is infinite or NaN).
```
Comment (keryell) [inline test_plans/individual-functions.asciidoc:261]: > to provide correct undefined return value behavior

Could you clarify?

**Review by keryell — APPROVED** [review]
Thanks.

**Comment by bader** [comment]
@keryell, do you want me to merge this PR as is or wait for @Melirius to apply suggested changes?

File: test_plans/individual-functions.asciidoc (hunk 2 above)
Comment (Melirius) [reply test_plans/individual-functions.asciidoc:261]: Definition in the Table 179 states "If x or y are infinite or NaN, the return values are undefined." Current reference implementation does not satisfy this requirement.

**Pull request merged**